packages/mux-player/src/template.ts

```diff
@@ -125,9 +133,10 @@ export const content = (props: MuxTemplateProps) => html`
     ${
       isLiveOrDVR(props)
         ? html`<button
+            disabled="${!hasSrc}"
```

Comment:
The issue is that there should be a difference between disabled no source and not-clickable because we're in the live window. Since, currently, the "in live window" styles are applied on aria-disabled. So, my current workaround was to leave current styles alone and target disabled+aria-disabled as the no-src option, to minimize changes in here and the CSS. However, maybe we want a separate attribute that indicates that this is in the live window separate from disabled/aria-disabled?

Reply:
we can probably hold off on that since we're anticipating an "official" implementation soon-ish.

Reply:
so, keep as is and fix it later?

Reply:
yeah keeping it as is is fine for now